public/static/js/projects.js

```diff
@@ -1378,16 +1378,34 @@ mciModule.controller(
 mciModule.directive("adminNewProject", function () {
   return {
     restrict: "E",
-    template: '<div class="row">' +
-      '<div class="col-lg-12">' +
-      "Enter project name " +
-      '<form style="display: inline" ng-submit="addProject()">' +
-      '<input type="text" id="project-name" placeholder="project name" ng-model="newProject.identifier">' +
-      ' <input type="checkbox" id="copy-project" ng-model="newProject.copyProject">' +
-      " Duplicate current project" +
-      "</form>" +
-      '<button type="submit" class="btn btn-primary" style="float: right; margin-left: 10px;" ng-click="addProject()">Create Project</button>' +
-      "</div>" +
-      "</div>",
+    template: '' +
+      '<div class="row">' +
+        '<div class="col-lg-12">' +
+         "Enter project identifier " +
+          '<input type="text" id="project-name" placeholder="project name" ng-model="newProject.identifier">' +
+        "</div>" +
+      '</div>' +
+      '<div class="row">' +
+        '<div class="col-lg-12">' +
+          "Optionally enter immutable project ID " +
```

Comment:
Do we want to maybe add a warning to only do this if they have a specific reason to?

Reply:
Yeah i was hoping that "Evergreen internally" was strong enough but it's probably worth being more clear